README.md

```diff
@@ -35,24 +35,25 @@ To use this sample, you will need :
 
 ## Register the application
 
-1. Sign into the [App Registration Portal](https://apps.dev.microsoft.com/) using either your personal or work or school account.
+1. Sign in to the [Azure Portal](https://portal.azure.com/) to register an application.
 
-1. Choose **Add an app**.
+1. If your account gives you access to more than one tenant, select your account in the top right corner, and set your portal session to the desired Azure AD tenant.

```

Comment:
(click on **Switch directory** and then choose the tenant)

Reply:
Thanks for the review. I'm following the recommendation from Lesia and Celeste to be consistent with the Docs article.